### PR TITLE
EZP-25822: Use subitem list item view

### PR DESCRIPTION
**Pull request by StephaneDiot**
jira: https://jira.ez.no/browse/EZP-25822
## Description

Goal here is to use ez-subitemlistitemview to render the items displayed in the search list.

### CODE REVIEW

**Comment by StephaneDiot** [comment]
ping @dpobel

File: Resources/public/js/views/ezsearch-searchlistview.js [hunk 1]
```diff
@@ -8,37 +8,96 @@ YUI.add('ezsearch-searchlistview', function (Y) {
...
+        _renderItems: function () {
+            var contentNode = this.get('container').one('.ezsearch-searchlist-content'),
+                ItemView = this.get('itemViewConstructor');
 
             if ( !this.get('searchResultList') ) {
-                return this.get('searchResultList');
+                return;
             }
-
+            this._destroyItemViews();
```
Comment (dpobel) [inline Resources/public/js/views/ezsearch-searchlistview.js]: there's nothing to destroy here but indeed this method should be called from Y.eZSearch.SearchListView `destructor`

File: Resources/public/js/views/ezsearch-searchlistview.js [hunk 2]
```diff
@@ -8,37 +8,96 @@ YUI.add('ezsearch-searchlistview', function (Y) {
...
             this.get('searchResultList').forEach(function (struct) {
-                searchResultItems.push({
-                    content: struct.content.toJSON(),
-                    contentType: struct.contentType.toJSON(),
-                    location: struct.location.toJSON(),
+                var view = new ItemView({
+                    displayedProperties: this.get('displayedProperties'),
```
Comment (dpobel) [inline Resources/public/js/views/ezsearch-searchlistview.js]: missing an indentation level

File: Resources/public/js/views/ezsearch-searchlistview.js [hunk 3]
```diff
@@ -8,37 +8,96 @@ YUI.add('ezsearch-searchlistview', function (Y) {
...
+                    identifier: identifier,
+                };
             }, this);
-            return searchResultItems;
         },
 
     }, {
         ATTRS: {
             searchResultList: {},
             searchResultItems: {},
```
Comment (dpobel) [inline Resources/public/js/views/ezsearch-searchlistview.js]: you forgot to remove that attribute in the previous change

**Comment by dpobel** [comment]
+1

**Pull request merged**